apps/condo/domains/billing/schema/BillingIntegration.js

```diff
@@ -28,23 +30,11 @@ const BillingIntegration = new GQLListSchema('BillingIntegration', {
             isRequired: true,
         },
 
-        shortDescription: {
```

Comment:
Why do you want to move these fields? Do you want to reuse it?

Reply:
Yes, we might reuse it after some time. Cant tell for sure untill I see makety